src/main/java/util/Utils.java

```diff
@@ -100,4 +100,18 @@ public static float constrain (float value, float min, float max) {
     public static float lerp (float start, float end, float amt) {
         return (1 - amt) * start + amt * end;
     }
+
+    /**
+     * Shifts a part of the array to overwrite the given region (fromIndex to toIndex)
+     * @param array the array from which the region has to be removed
+     * @param fromIndex start of the region
+     * @param toIndex end of the region
+     * @return
+     */
+    public static int shiftOverwrite(float[] array, int fromIndex, int toIndex) {
```

Comment:
maybe make clear, that toIndex is exclusive and fromIndex is inclusive

Reply:
Added to the javadoc comment. With some fancy interval notation too :D.